### ISSUE TITLE
Language

### ISSUE THREAD

**Issue by Flop2006** [ggevgg/weather-widget#1]
Changing of the lang flag in index.js causes no change.

**Comment by ggevgg** [comment]
At the present time changing of the lang flag influences only the language of API response api.openweathermap.org (field weather.description). Thank you for your note.

**Comment by Flop2006** [comment]
Yes, but if i change it in the app.js it works. But not when i change it in the index.js

**Comment by Flop2006** [comment]
Oh i see in the actual DEMO on jqueryscript.net there isn´t the lang flag in the index.js anymore.
